plugins/mcreator-core/triggers/entity_pre_hurt.json

```diff
@@ -28,11 +28,16 @@
       "name": "amount",
       "type": "number"
     },
+    {
+      "name": "immediatesourceentity",
+      "type": "entity"
+    },
     {
       "name": "damagesource",
       "type": "damagesource"
     }
   ],
   "cancelable": "true",
   "side": "server"
-}
\ No newline at end of file
+}

```

Comment:
Those changes are unrelated. Please use proper IDE to do changes, not GitHub online editor.